File: wit-wasi-java/src/test/java/com/example/wasm/HostWasmRunnerTest.java
package com.example.wasm;

import org.junit.Test;
import static org.junit.Assert.assertEquals;
import java.io.IOException;

public class HostWasmRunnerTest {
    @Test
    public void testAdd() throws IOException {
        HostWasmRunner runner = new HostWasmRunner("examples/java-jvm/example.wasm");
        assertEquals(10, runner.add(4, 6));
    }
}

File: wit-wasi-kotlin/src/test/kotlin/com/example/wasm/HostWasmRunnerTest.kt
package com.example.wasm

import kotlin.test.Test
import kotlin.test.assertEquals

class HostWasmRunnerTest {
    @Test
    fun testAdd() {
        val runner = HostWasmRunner("examples/kotlin-jvm/example.wasm")
        assertEquals(5, runner.add(2, 3))
    }
}